tools/github-release.ts

```diff
@@ -8,43 +8,35 @@ type Options = {
   owner: string
   repo: string
   releaseNotes: string
-  downloadLinks: string
-  betaRelease: boolean
-  dryRun: boolean
+  prerelease: string
 }
 
 const githubRelease = async (
   platform: string,
   newVersionName: string,
   newVersionCode: string,
-  { deliverinoPrivateKey, owner, repo, releaseNotes, downloadLinks, betaRelease, dryRun }: Options,
-) => {
+  { deliverinoPrivateKey, owner, repo, releaseNotes, prerelease }: Options,
+): Promise<void> => {
   const versionCode = parseInt(newVersionCode, 10)
   if (Number.isNaN(versionCode)) {
     throw new Error(`Failed to parse version code string: ${newVersionCode}`)
   }
 
-  const releaseName = `[${platform}${betaRelease ? ' beta release' : ''}] ${newVersionName} - ${versionCode}`
-  console.warn('Creating release with name ', releaseName)
-
-  const betaMessage = betaRelease ? 'This release is only delivered to beta and not yet visible for users.\n\n' : ''
-
-  const body = `${betaMessage}${JSON.parse(releaseNotes)}${downloadLinks ? `\nArtifacts:\n${downloadLinks}` : ''}`
-  console.warn('and body ', body)
-
-  if (dryRun) {
-    return
-  }
-
+  const releaseName = `[${platform}] ${newVersionName} - ${versionCode}`
+  const body = releaseNotes
   const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
 
-  await appOctokit.repos.createRelease({
+  const release = await appOctokit.repos.createRelease({
     owner,
     repo,
     tag_name: tagId({ versionName: newVersionName, platform }),
+    prerelease: prerelease === 'true',
+    make_latest: platform === 'android' ? 'true' : 'false',
     name: releaseName,
     body,
   })
+
+  console.log(release.data.id)
```

Comment:
❓  Is this still test code? If not, maybe elaborate in the message what that id is?

Reply:
No it is not. This is the output of the command, which returns this id.